src/components/button/Index.module.css

```diff
@@ -0,0 +1,64 @@
+.primary {
+  width: 69px;
+  height: 40px;
+  padding: 12px, 16px, 12px, 16px;
+  border-radius: 8px;
+  gap: 10px;
+
+  border: 1px;
+  cursor: pointer;
+  background-size: 50%;
+
+  box-shadow: 0px 1px 3px 0px #00000014;
+  transition: transform 0.3s ease;
+  display: none;
+  background-color: rgba(21, 93, 251, 1);
+  color: rgba(255, 255, 255, 1);
+}
+
+.primary:hover {
+  background-color: rgba(73, 130, 255, 1);
+}
+
+.primary:focus {
+  border: 3px solid rgba(21, 93, 251, 0.25);
+}
+
+.secondary {
+  border-radius: 8px;
+  border: 1px;
+  cursor: pointer;
+
+  background-size: 50%;
+
+  background-color: #ffffff;
+  border-color: #ffffff;
+  box-shadow: 0px 1px 3px 0px #00000014;
+  transition: transform 0.3s ease;
+  display: none;
+}
```

Comment:
Can you make sure that the styles that are common in both the primary and secondary button are put together. So like: 

```
.primary, 
.secondary {
--- common styles ---
} 

.primary {
--- specific styles ---
} 

.secondary {
--- specific styles --- 
}
```